src/APIManager.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+module.exports = class APIManager {
+    constructor(instance) {
+        this.instance = instance;
+        this.apiFunctions = {};
+
+        // record fn to send data to instance
+        this.registerFunction(
+            'sendData',
+            (json) => {
+                this.instance.sendEvent(json);
+            },
+            'send data to WS messages of player, must be a JSON object. Example: {type: "MOUSE_PRESS", x: 100, y: 100}',
+        );
+
+        // record fn to get registered functions
+        this.registerFunction(
+            'getRegisteredFunctions',
+            () => this.listRegisteredFunctions(),
```

Comment:
Could we have the same name in both places? Either `getRegisteredFunctions` or `listRegisteredFunctions`, but not both at the same time?